src/css/home.css

```diff
@@ -0,0 +1,154 @@
+* {
```

Comment:
Find and delete all the redundant classes here that are already defined in theme.css, header.css, and footer.css